packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -936,12 +905,13 @@ export function applyThreadUpdates<M extends BaseMetadata>(
   updates.newThreads.forEach((thread) => {
     const existingThread = updatedThreads[thread.id];
 
-    // If the thread already exists, we need to compare the two threads to determine which one is newer.
+    // If a thread already exists and it's never, don't touch it
```

Comment:
```suggestion
    // If a thread already exists and it's never, don't touch it
```
not sure what this sentence means?

Reply:
LOL, me neither 😂